src/callbacks.jl

```diff
@@ -31,6 +31,32 @@ function callback_value(cb_data, x::VariableRef)
     )
 end
 
+"""
+    callback_value(cb_data, expr::GenericAffExpr)
+
+Return the primal solution of an affine expression inside a callback by getting
+the value for each variable appearing in the expression.
+
+`cb_data` is the argument to the callback function, and the type is dependent on
+the solver.
+"""
+function callback_value(cb_data, expr::GenericAffExpr)
```

Comment:
We can merge the two methods since they are identical

Reply:
yup done